### PR TITLE
Add contribution guidelines

### PR DESCRIPTION
**Pull request by adam-be**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +23,25 @@ The content of this document is discussed at the
...
+Example: Add support for new attribute X on Y (fixes: #123).  
+Avoid: Fix for #123.
+
+#### Linked Names
+Make names of interfaces, enums and other identifiers clickable like this ```<code><a>MediaStreamTrack</a></code>```.
+
+#### JSEP References
+Reference specific sections of JSEP where possible. The syntax looks like this:  
+```<span data-jsep="initial-offers initial-answers">[[!JSEP]]</span>```  
+Section aliases, to be used as data-jsep values, are found in [jsep-mapping/map.json](jsep-mapping/map.json). Information on how to update the map is found in [jsep-mapping/README.md](jsep-mapping/README.md).
```
Comment (adam-be) [inline README.md]: I'm not sure why the syntax highlighting is messed up here. It might be a github bug.

**Comment by vivienlacourba** [comment]
Hi @adam-be,

Shouldn't this be in a separate `CONTRIBUTING.md` file?
It would make those guidelines more visible (eg they would be integrated automatically in the page to submit new pull request).

See [Github's Contributing Guidelines  blog post](https://github.com/blog/1184-contributing-guidelines) for details.

File: README.md [hunk 2]
```diff
@@ -23,3 +23,25 @@ The content of this document is discussed at the
 mailing list.
 
 [RTCWeb IETF Working Group](https://tools.ietf.org/wg/rtcweb/)
+
+### Contribution Guidelines
+Since we are a lot of people contributing to the specification, we have defined a few guidelines. Please follow them and we will be able to review your PR a lot faster when we don't have to point out style and other non-technical issues. Thank you.
+
```
Comment (alvestrand) [inline README.md]: Shouldn't we also mention the membership-checking IPR bot and what you need to do to satisfy it here?

**Comment by adam-be** [comment]
> Shouldn't this be in a separate CONTRIBUTING.md file?

Yes it should. I'll do an update shortly.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -6,3 +9,22 @@ in the relevant W3C Working Group or make a non-member patent licensing commitme
 If you are not the sole contributor to a contribution (pull request), please identify all contributors in the 
 commit message. In the commit, please include on a new line,
 <pre>Contributors: +@githubusername1, +@githubusername2, ...</pre>
+
+### 80 Characters
```
Comment (alvestrand) [inline CONTRIBUTING.md]: Need to change this to reflect that you should do "make tidy" instead (as a separate commit).

**Comment by adam-be** [comment]
Someone informed me that Travis isn't enforcing line length on this document so we might want to revert the last commit.

**Pull request merged**